packages/extension-polkagate/src/popup/staking/stakingInfo/index.tsx

```
@@ -26,19 +26,21 @@ interface Props {
 
 function StakingInfo ({ selectedPosition, setSelectedPosition }: Props): React.ReactElement {
   const { t } = useTranslation();
+  const navigate = useNavigate();
   const { availableBalance, decimal, genesisHash, tokenSymbol } = selectedPosition;
   const poolConsts = usePoolConst(genesisHash);
   const stakingConsts = useStakingConst(genesisHash);
   const _decimal = decimal || stakingConsts?.decimal || 1;
 
   const eraLength = remainingTime(poolConsts?.eraLength?.toNumber() ?? 0);
   const handleClose = useCallback(() => setSelectedPosition(undefined), [setSelectedPosition]);
+  const goStaking = useCallback(() => navigate('/pool/' + genesisHash + '/stake') as void, [genesisHash, navigate]);
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)